src/pybind/mgr/orchestrator.py

```diff
@@ -183,19 +183,21 @@ def wait(self, completions):
         """
         raise NotImplementedError()
 
-    def add_host(self, host):
-        # type: (str) -> WriteCompletion
+    def add_host(self, host, role=None):
+        # type: (str, str) -> WriteCompletion
         """
         Add a host to the orchestrator inventory.
         :param host: hostname
+        :param role: Role to performn (osd, mon, mgr)
```

Comment:
So, in the Ansible world, a host can only have one single role?

Reply:
No. A host is able to has different roles and therefore included in different Ansible Inventory groups